packages/react-ui/storybook/stories/molecules/Alert.stories.js

```diff
@@ -0,0 +1,248 @@
+import React from 'react';
+import { Box, Button, Typography, Badge, capitalize, styled } from '@mui/material';
+import { Standalone, ThinContainer } from '../../utils/storyStyles';
+import Alert from '../../../../react-ui/src/components/molecules/Alert';
+import AutoAwesomeOutlinedIcon from '@mui/icons-material/AutoAwesomeOutlined';
+
+const title = 'This is a title';
+const inlineText = 'This is a Inline content alert';
+const blockText =
+  'This is a Block content alert. Use the Block content variant for alerts with a long paragraph, more than two lines, or a title and body text.';
+
+const options = {
+  title: 'Molecules/Alert',
+  component: Alert,
+  argTypes: {
+    title: {
+      control: {
+        type: 'text'
+      }
+    },
+    severity: {
+      control: {
+        type: 'select',
+        options: ['neutral', 'info', 'success', 'warning', 'error']
+      }
+    },
+    content: {
+      control: {
+        type: 'select',
+        options: ['inline', 'block']
+      }
+    }
+  },
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/nmaoLeo69xBJCHm9nc6lEV/CARTO-Components-1.0?node-id=1534%3A36257'
+    },
+    status: {
+      type: 'inDevelopment'
```

Comment:
please, mark this as ReadyToReview, I'm going to upload to internal storybook so design team can validate the implementation

Reply:
:+1: Done in [this commit](https://github.com/CartoDB/carto-react/pull/698/commits/aa3c254878d7b3a770bbcc92876133e9b043b6f2)